infrastructure/docker/builder/Dockerfile

```diff
@@ -2,6 +2,6 @@ FROM golang:1.17.3-alpine3.15 as golang
 FROM docker:20.10.16-dind-alpine3.15
 COPY --from=golang /usr/local/go/ /usr/local/go/
 ENV PATH /usr/local/go/bin:$PATH
-RUN apk add --no-cache libgit2 libgit2-dev go protoc make pkgconfig build-base yarn git
+RUN apk add --no-cache libgit2 libgit2-dev go protoc make pkgconfig build-base yarn git tar
```

Comment:
why do we need `tar` now?

Reply:
Please explain it in a comment.
If it's a execution-plan dependency, it should appear in the execution-plan readme

Reply:
This is a actions/cache dependancy. it needs gnu tar in the builder image and not busybox tar.

I'll add that as a readme in execution plan.
https://github.com/actions/toolkit/issues/634